android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -294,6 +302,26 @@ public void setDisableFocus(final ReactExoplayerView videoView, final boolean di
         videoView.setDisableFocus(disableFocus);
     }
 
+    @ReactProp(name = PROP_BACK_BUFFER_DURATION_MS, defaultInt = 0)
+    public void setBackBufferDurationMs(final ReactExoplayerView videoView, final int backBufferDurationMs) {
+        videoView.setBackBufferDurationMs(backBufferDurationMs);
+    }
+
+    @ReactProp(name = PROP_CONTENT_START_TIME, defaultInt = 0)
```

Comment:
Great, this is a very good feature! :) 
But I think we need to do a bigger change to have a safe API...

As you see there is an uggly postDelayed in initializePlayer.
I think we should addapt the APi to avoid this postdelay.

Its goal is to be able to receive all initialization props before really starting playback.

To avoid this there is a big api change to:
I really think we shall merge all props to startup a content in source object.

I identify: DRM config, sideloaded subtitles track, buffering parameters, useTextureView,  and this content start time !

Notice that I also implement a contentStartTime in my fork, This is a good feature !

Reply:
I created an issue for this https://github.com/react-native-video/react-native-video/issues/2693 feel free to comment any implementation details you feel could be useful to include in the issue.